_data/cryptocurrencies.yml

```diff
@@ -176,6 +176,13 @@ websites:
       software: Yes
       doc: https://localbitcoins.blogspot.com.au/2013/05/using-two-factor-authentication-on.html
 
+    - name: SatoshiTango
+      url: https://satoshitango.com/
+      img: satoshitango.png
+      tfa: Yes
+      software: Yes
+      doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
```

Comment:
@jamcat22 I've seen sites with similar doc links, but it's up to you guys. But, creating a note will be more cleaner than this, for sure.